usr/iscsi/conn.c

```diff
@@ -95,6 +95,7 @@ void conn_close(struct iscsi_connection *conn)
 
 	conn->closed = 1;
 
+	eprintf("[Sumitk] ep close\n");
```

Comment:
Remove all messages starting from Sumitk from this file.

Reply:
Done